include/boost/geometry/index/detail/serialization.hpp

```diff
@@ -65,10 +64,10 @@ class serialization_storage
     }
     T * address()
     {
-        return static_cast<T*>(m_storage.address());
+        return static_cast<void*>(&m_storage);
```

Comment:
@awulkiew is this fine?

Reply:
`std::aligned_storage` is deprecated in C++23. AFAIR boost version is superior to the std one but I can't remember why besides the fact that it doesn't force us to cast address of the storage. A proposed replacement is something like this:
```
alignas(T) std::byte storage[sizeof(T)];
```
but it requires C++17 and would require careful testing. So I'd prefer to keep `boost::aligned storage` for now.

Reply:
Btw, here https://en.cppreference.com/w/cpp/types/aligned_storage the required use of `std::launder` is mentioned. This change can introduce some unwanted effects. I guess I strongly suggest to keep the old version. :)